.gitignore

```diff
@@ -0,0 +1,2 @@
+example-shoppe-ios/example-shoppe-ios.xcodeproj/project.xcworkspace/contents.xcworkspacedata
+example-shoppe-ios/example-shoppe-ios.xcodeproj/project.xcworkspace/xcuserdata/harold.xcuserdatad/UserInterfaceState.xcuserstate
```

Comment:
should we ignore the whole xcuserdata folder? I assume the username would be different if I clone the project

Reply:
Thanks so much I appreciate you bringing that up. Actually it would be beneficial for this project to leave this in tact, so I will actually remove these from the gitignore.